chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -249,14 +249,14 @@ impl PartialWitnessActor {
     }
 
     // Function to generate the parts of the state witness and return them as a tuple of chunk_validator and part.
-    fn generate_state_witness_parts(
+    fn generate_and_send_state_witness_parts(
         &mut self,
         epoch_id: EpochId,
         chunk_header: &ShardChunkHeader,
         witness_bytes: EncodedChunkStateWitness,
         chunk_validators: &[AccountId],
         signer: &ValidatorSigner,
-    ) -> Result<Vec<(AccountId, PartialEncodedStateWitness)>, Error> {
+    ) -> Result<(), Error> {
```

Comment:
you might ave to wait for Akhi's [PR](https://github.com/near/nearcore/pull/12645) to land